microsite/data/plugins/aws-app-development.yaml

```
@@ -0,0 +1,10 @@
+---
+title: AWS App Developer Tools
+author: Amazon Web Services
+authorUrl: https://aws.amazon.com/
+category: Infrastructure
+description: Create and manage AWS Apps within Backstage```
```

```suggestion
description: Create and manage AWS Apps within Backstage
```
Sorry but the three backticks here were unintentional

---

no problem. revised.